src/extension/views/doc-source/js/ui.js

```diff
@@ -237,8 +237,9 @@ const load = async () => {
       copyButton.classList.remove('copied');
     }, 2000);
 
-    sampleRUM('sidekick:copydocsource', {
-      source: tab.url,
+    sampleRUM('click', {
+      source: 'sidekick',
+      target: 'copydocsource',
```

Comment:
Should we change this to past tense to match the others, e.g. `doc-source-opened`?